keps/sig-node/4580-deprecate-kubelet-runonce/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Deprecate & remove Kubelet RunOnce mode
+kep-number: 4580
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-node
+participating-sigs:
+status: provisional
+creation-date: 2024-04-17
+reviewers:
+  - "@tallclair"
+  - "@dchen1107"
+approvers:
+  - TBD
+
+see-also:
+
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.31"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.31"
+  beta: "v1.32"
+  stable: "v1.34"
```

Comment:
According to [Deprecating a flag or CLI](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli), I don't think we can deprecate the runonce flag in two versions, so follow [Deprecating a flag or CLI](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli) and extend the beta time

Reply:
You can propose the timeline in the document, but fill in these stages only when actually reaching a particular one. In the past we've had those fields filled in and the feature wasn't progressing confusing users.

Reply:
Consider using comments to capture the intent without making (what appears to be) a promise.